### PR TITLE
i18n(it): update `guides/authoring-content`

### PR DESCRIPTION
**Pull request by itsmatteomanf**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

Updates minor translation issues of `guides/authoring-content.md` hoping it satisfies the tracker.

### CODE REVIEW

File: docs/src/content/docs/it/guides/authoring-content.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ description: Come utilizzare i link automatici di Starlight
 
 ## Introduzione
 
-Posso collegarmi alla [conclusione](#conclusione) che si trova più in basso.
+Posso collegarmi alla [mia conclusione](#conclusione) che si trova più in basso.
```
Comment (ematipico) [inline docs/src/content/docs/it/guides/authoring-content.md:83]: Does the original content contain this word?

**Review by ematipico — APPROVED** [review]

File: docs/src/content/docs/it/guides/authoring-content.md (hunk 1 above)
Comment (itsmatteomanf) [reply docs/src/content/docs/it/guides/authoring-content.md:83]: It actually does...

https://github.com/withastro/starlight/blob/def71050fbdb9ae514b0a7c24ed2455f055fe6bd/docs/src/content/docs/guides/authoring-content.md?plain=1#L83C67-L83C67

**Review by delucis — COMMENTED** [review]
Grazie per i correzioni @itsmatteomanf.

Guardando https://github.com/withastro/starlight/commit/02c8b54a3d3d01113ad7428bf564b77f137ded1e vedo che è stato aggiunto il collegamento verso `https://docs.astro.build/it/guides/assets/` nella sezione “Immagini”. Potresti aggiornare questo?

**Comment by itsmatteomanf** [comment]
@delucis fatto e sincronizzato con main.

File: docs/src/content/docs/it/guides/authoring-content.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Puoi evidenziare `codice in linea` con apici inversi.
 
 ## Immagini
 
-Le immagini in Starlight utilizzano l'ottimizzazione degli asset di Astro.
+Le immagini in Starlight utilizzano [l'ottimizzazione degli asset di Astro](https://docs.astro.build/en/guides/assets/).
```
Comment (delucis) [inline docs/src/content/docs/it/guides/authoring-content.md:32]: We can use the Italian docs page for this link:

```suggestion
Le immagini in Starlight utilizzano [l'ottimizzazione degli asset di Astro](https://docs.astro.build/it/guides/assets/).
```

**Review by delucis — APPROVED** [review]
Looks great — thanks everyone 🙌

**Pull request merged**